src/components/dropdown.js

```diff
@@ -47,5 +52,9 @@ class FlavorForm extends React.Component {
     }
   }
 
+  const stateMappings = state => ({})
+  const actionMappings = {
+      dropDown
+  }
 
-export default FlavorForm;
\ No newline at end of file
+export default connect(stateMappings, actionMappings)(FlavorForm);
```

Comment:
Might want to update the class name from `FlavorForm` to `DropDown` at some point :)
Same at the top of this file.